packages/back-end/types/feature.d.ts

```diff
@@ -49,7 +49,7 @@ export interface FeatureInterface {
 }
 type ScheduleRule = {
   timestamp: string | null;
-  enableFeature: boolean;
+  status: "enabled" | "disabled";
```

Comment:
Are there ever going to be more than 2 statuses here?  If not, this seems like it should be a boolean

Reply:
Updated to 

```
type ScheduleRule = {
   timestamp: string | null;
   scheduleRuleEnabled: boolean;
}